### ISSUE TITLE
bug: alert_sources AWS on lacework_alert_rule does not match with the Lacework console.

### ISSUE THREAD

**Issue by payamzowghi** [lacework/terraform-provider-lacework#554]
**Describe the bug**
alert_sources AWS on lacework_alert_rule does not match with the Lacework console.

**To Reproduce**
1. terraform plan To create alert rule(after deploy it shows as `Aws` not `AWS` ):
```
resource "lacework_alert_rule" "example" {
  name                = "example"
  description         = "xxxxxxxxxxxx"
  alert_channels      = [lacework_alert_channel_pagerduty.payfacto_og_high.id]
  severities          = ["Critical"]
  alert_subcategories = ["Registry","App","Cloud","File","Machine","User","Platform","K8sActivity","SystemCall"]
  alert_categories    = ["Policy"]
  alert_sources       = ["Aws"]

}


~ resource "lacework_alert_rule" "example" {
      ~ alert_sources           = [
          - "AWS",
          + "Aws",
        ]
        id                      = "xxxxx_4206A740F414CDCAE6A9F20B7A00987E69E8323FDF88C3F"
        name                    = "example"
        # (9 unchanged attributes hidden)
    }

```

**Expected behavior**
No drift.

**Screenshots**
<img width="469" alt="Screen Shot 2023-09-10 at 2 58 28 AM" src="https://github.com/lacework/terraform-provider-lacework/assets/25571762/534f2600-8b2f-45ac-b245-fcedb476a824">

**Please complete the following information):**
 - Provider Version [v1.14.0]

**Comment by afiune** [comment]
Closed by https://github.com/lacework/go-sdk/pull/1376

**Comment by afiune** [comment]
Actually this hasn't been solved. Apologies, we are working on this ticket now.

**Comment by PengyuanZhao** [comment]
@payamzowghi This should be fixed in [v1.15.1](https://github.com/lacework/terraform-provider-lacework/pull/563). Please check it out.
